src/lib/Utils.js

```diff
@@ -3,32 +3,53 @@ import moment from 'moment';
 import { Types } from '../constants';
 import * as Locales from '../locales';
 
-// Remap Locale keys to match moment locales
-const _locales = _.mapKeys(Locales, (v, k) => _.toLower(k));
-
 // Mustache style templating is easier on the eyes
 _.templateSettings.interpolate = /{{([\s\S]+?)}}/g;
 
 // Set locale
-// Locale lookup for date name strings are based on moment
-let _locale = {};
-let _fallbackLocale = _.get(_locales, 'enus');
+// Locale lookup for date name strings are based on moment.
+// Romcal define at least the default 'en' language as a fallback.
+// If a region is specified in the locale ('xx-XX'), romcal will
+// automatically manage a graceful fallback to its base language ('xx'), if it exists in 'src/locales'.
+// We get then a cascade fallbacks: region ('xx-XX') -> base language ('xx') -> default 'en'
+// E.g.: If a string is missing in 'fr-CA', it will try to pick it in 'fr', and then in 'en'.
+const _fallbackLocaleKey = 'en';
+let _combinedLocale;
+let _locales;
 let setLocale = key => {
 
+  // When setLocale is called, redefines this vars to defaults
```

Comment:
What does do the redefinition? As I understand it, the `setLocale`; is such case it is sufficient to use _**it** redefines_, otherwise replace _it_ with proper subject that does the redefinition.